.ci/ci.sh

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env bash
 
-[ -d "$HOME/.ccache" ] && sudo chown -R $USER: $HOME/.ccache
+if [ ! $NOTROOT = 1 ]; then
+  SUDO=sudo
+fi
```

Comment:
```suggestion
if [[ $NOTROOT != 1 ]]; then
  SUDO=sudo
fi
```
since we are using bash, it is better to use the double `[[` version so we do not have to worry about quoting the variable. This might also fix the issue with FFmpeg and gstreamer

Reply:
modified and it works, thank you for help!